public/js/sserve/controllers.js

```diff
@@ -1,895 +1,984 @@
-                                        _
-                                            .each(
-                                                result.data.entity,
-                                                function(
-                                                    migrated) {
-                                                    migrated.status.data = prepareReport(migrated.status.data);
-                                                })
-                                        result = transformMigrated(result)
-                                        $scope.migratedProjects = _
-                                            .sortBy(
-                                                result.data.entity,
-                                                'site_name');
-                                        // this poll has
-                                        // different data
-                                        // than the last one
-                                        if (!angular
-                                            .equals(
-                                                $scope.migratedProjects,
-                                                $scope.migratedProjectsShadow)) {
-                                            $log
-                                                .info('migrated has changed - call a function to update projects panel');
-                                            // update
-                                            // project panel
-                                            // based on new
-                                            // data
-                                            $log
-                                                .info('Updating projects panel based on COMPLETED migrations poll');
-                                            updateProjectsPanel(
-                                                $scope.migratedProjects,
-                                                'migrated')
-                                        }
-                                        $scope.migratedProjectsShadow = _
-                                            .sortBy(
-                                                result.data.entity,
-                                                'site_name');
-                                        // clear error
-                                        // condition if any
-                                        $scope.migratedProjectsError = false;
-                                    } else {
-                                        // declare error and
-                                        // show error
-                                        // message in UI
-                                        $scope.migratedProjectsError = true;
-                                    }
-
-                                }
-                            });
                 }
+              });
+        }
 
-    
```

Comment:
Are "You have" and "moment().valueOf()" used as placeholder/default value for userId and consentTime respectively?